app/assets/stylesheets/variables.scss

```diff
@@ -0,0 +1,14 @@
+$transparent: rgba(0, 0, 0, 0);
+
+// Hack Club gradient colors
+$red: #ff4b55;
+$purple: #d44e74;
+$white: #ffffff;
+$blue: #4ca9ff;
+$green: #0be9d1;
+
+// shadow colors
+$shadow-1: rgba(0, 0, 0, 0.12);
+$shadow-2: rgba(0, 0, 0, 0.22);
+$shadow-3: rgba(0, 0, 0, 0.24);
+$shadow-4: rgba(0, 0, 0, 0.25);
```

Comment:
This comment is out-dated by https://github.com/hackclub/lecture-hall/pull/41/commits/d90e1c885030592f47a5fbdfcdcd63c23416a155